README.rst

```diff
@@ -56,11 +56,11 @@ Listed below are the actively maintained release branches along with their lates
 patch release, corresponding image pull tags and their release notes:
 
 +---------------------------------------------------------+------------+------------------------------------+----------------------------------------------------------------------------+
-| `v1.17 <https://github.com/cilium/cilium/tree/v1.17>`__ | 2025-02-04 | ``quay.io/cilium/cilium:v1.17.0``   | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.17.0>`__ |
+| `v1.17 <https://github.com/cilium/cilium/tree/v1.17>`__ | 2025-02-04 | ``quay.io/cilium/cilium:v1.17.1``   | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.17.1>`__ |
```

Comment:
The date wasn't updated here?

Reply:
Possibly related to the column misalignment(?) https://github.com/cilium/cilium/pull/37728 (needs rebase)